krm-functions/lib/kptfile/v1/kpt_test.go

```diff
@@ -0,0 +1,332 @@
+/*
+Copyright 2023 Nephio.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+import (
+	"testing"
+
+	kptv1 "github.com/GoogleContainerTools/kpt/pkg/api/kptfile/v1"
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestParseObject(t *testing.T) {
```

Comment:
usually the name should be "Test" +"name of function", i.e in this case it would be "Test"+"ParseKubeObject" => TestParseKubeObject

Reply:
fixed